docs/user-guide/plugins/bitbucket-runner-trigger.md

```diff
@@ -0,0 +1,60 @@
+# BitBucket Runner Trigger
```

Comment:
Except the variables, replace all instances of `BitBucket` with `Bitbucket` in our doc.
Again, we need to request the devs to fix it in the plugin name and its subtext.

Reply:
changed `BitBucket` to `Bitbucket` in doc